Documentation/ExtensionArchitecture/Extbase/Reference/Controller/TypeConverter.rst

```diff
@@ -0,0 +1,57 @@
+.. include:: /Includes.rst.txt
+.. index::
+   Extbase; Type converters
+.. _extbase_Type_converters:
+
+===============
+Type converters
+===============
+
+Type converters are commonly used when it is necessary to convert from one type
+into another. They are usually applied in the controller in the method
+:php:`initialize<actionName>Action()` method.
+
+For example a date might be given as string in some language,
+:php:`"Freitag, 7. Oktober 2022"` or as UNIX time stamp: :php:`1665159559`.
+Your action method, however, expects a :php:`\Date` object. Extbase tries to
+match the data coming from the frontend automatically.
+
+When matching the data formats is expected to fail you can use one of type
+converters provided by Extbase or implement a type converter yourself
+by extending :php:`TYPO3\CMS\Extbase\Property\TypeConverter\AbstractTypeConverter`.
+
+You can find the type converters provided by Extbase in the directory
+`EXT:extbase/Classes/Property/TypeConverter
+<https://github.com/TYPO3/typo3/tree/main/typo3/sysext/extbase/Classes/Property/TypeConverter>`__.
+
+Custom type converters
+======================
+
+..  versionchanged:: 12.0
+    Starting with TYPO3 12.0 a type converter does not have to be registered
+    via the now deprecated method :php:`\TYPO3\CMS\Extbase\Utility\ExtensionUtility::registerTypeConverter()`.
+    Remove calls to this method when dropping TYPO3 v11 support. It will be
+    removed with TYPO3 13. Register a type converter in your extension's
+    :file:`Services.yaml` instead.
+
+A custom type converter must extend the abstract class
```

Comment:
Nope, must implement the TypeConverterInterface or extend the AbstractTypeConverter.